users/views.py

```diff
@@ -1,3 +1,80 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import jwt
+import re
 
-# Create your views here.
+from django.views          import View
+from django.http           import JsonResponse
+
+from json.decoder          import JSONDecodeError
+
+from users.models          import User, UserInformation
+from my_settings           import SECRET_KEY, ALGORITHM
+
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data          = json.loads(request.body)
+            email         = data.get("email")
+            password      = data.get("password", None)
+            nickname      = data.get("nickname")
+            is_expert     = data.get("is_expert", False)
+            kakao_id      = data.get("kakao_id", None)
+            user_info     = User.objects.filter(email=email)
```

Comment:
```suggestion
            data      = json.loads(request.body)
            email     = data.get("email")
            password  = data.get("password", None)
            nickname  = data.get("nickname")
            is_expert = data.get("is_expert", False)
            kakao_id  = data.get("kakao_id", None)
```
필수인 것들은 key error가 나도록 수정해 주세요.

Reply:
네 수정 반영하겠습니다.